setup.py

```diff
@@ -108,6 +112,6 @@
     url="https://github.com/AllenCellModeling/aicsimageio",
     # Do not edit this string manually, always use bumpversion
     # Details in CONTRIBUTING.md
-    version="3.3.1",
+    version="3.3.0",
```

Comment:
version downgraded?

Reply:
Yea this is just branches not being merged and such. I am planning on have a big ol merge day where I merge the `feature/update-to-4.0-api` with `release-4.0` and then create the PR for `release-4.0` into `master`. The diffs aren't large but just haven't gotten around to it.

Also reminder, that really, versions don't matter until something is on `master` and we run `make prepare-release`